src/pages/api/dataverse/dataspace/[dataspaceId]/service/index.ts

```diff
@@ -11,20 +11,15 @@ const handler = (req: NextApiRequest, res: NextApiResponse): void => {
   }
 
   const dataspaceId = req.query.dataspaceId as string
-  const size = req.query.size as string
+  const size = req.query.size ? parseInt(req.query.size as string) : 10
```

Comment:
just a minor remark: ideally, it would be advisable to set an upper limit, to prohibit values that are too high.